docs/source/known-issues.rst

```diff
@@ -4,6 +4,20 @@ Known issues
 .. role:: cppkokkos(code)
     :language: cppkokkos
 
+CUDA
+====
+
+- When using a legacy NVIDIA GPU, Kokkos' initialization can report errors like:
```

Comment:
Is the term "legacy" here unambiguous?  In other words, maybe a simple table to show which Nvidia architectures / CUDA versions are / are not compatible with `cudamallocasync` and/or memory pools.

Reply:
It is not straightforward to do a table as it depends on both the architecture and the driver/OS configuration.
Perhaps we should remove *legacy*, the goal is just to reference, if you see this error, try this to solve it.